### PR TITLE
Improve server host command

### PR DESCRIPTION
**Pull request by Shawak**
Start the server with a random map if none specified

### CODE REVIEW

File: server/src/io/anuke/mindustry/server/ServerControl.java [hunk 1]
```diff
@@ -211,19 +211,25 @@ private void registerCommands(){
...
-
-            if(result == null){
-                err("No map with name &y'{0}'&lr found.", arg[0]);
-                return;
+                if(result == null){
+                    err("No map with name &y'{0}'&lr found.", arg[0]);
+                    return;
+                }
+            } else {
+                Array<Map> maps = Array.with(Vars.maps.customMaps().size == 0 ? Vars.maps.defaultMaps() : Vars.maps.customMaps());
```
Comment (Anuken) [inline server/src/io/anuke/mindustry/server/ServerControl.java]: `Array.with` should not be necessary here at all.

File: server/src/io/anuke/mindustry/server/ServerControl.java [hunk 2]
```diff
@@ -211,19 +211,25 @@ private void registerCommands(){
...
+                result = maps.get(new Random().nextInt(maps.length));
```
Comment (Anuken) [inline server/src/io/anuke/mindustry/server/ServerControl.java]: Use `maps.random()`, which just returns a random element.

File: server/src/io/anuke/mindustry/server/ServerControl.java [hunk 3]
```diff
@@ -211,19 +211,25 @@ private void registerCommands(){
             info("Stopped server.");
         });
 
-        handler.register("host", "<mapname> [mode]", "Open the server with a specific map.", arg -> {
+        handler.register("host", "[mapname] [mode]", "Open the server with a random or specific map.", arg -> {
```
Comment (Anuken) [inline server/src/io/anuke/mindustry/server/ServerControl.java]: This is not a very good description for this command. It should be:
`Open the server. Will default to survival and a random map if not specified.`

**Comment by Anuken** [comment]
Also, some word of advice: don't try to write Java or submit pull requests without importing the project into IntelliJ first. Even with the simplest changes, it won't be easy to make do without an IDE. Run pull requests before submitting them.

**Comment by Shawak** [comment]
Yeah I just didn't want to install Java on my machine. I got it running inside a docker image now, lemme rebase this.

**Comment by Shawak** [comment]
@Anuken that should do it

**Pull request merged**